### ISSUE TITLE
New package for inclusion - (monitoring) meteor-apm-server

### ISSUE THREAD

**Issue by afrokick** [Meteor-Community-Packages/organization#37]
## meteor-apm-server

> This project reduces the original Kadira APM to a single Meteor project and includes template MUP configuration to let you deploy to any remote server.

We use it in production last year, very stable.

## Links
* [Github repository](https://github.com/lmachens/meteor-apm-server)

## Current status of the project
 * Active
 * Maintained
 * Abandoned

 ## Reasoning
You can use MontiAPM instead. But some projects require its own infrastructure (without 3rd party). This package looks good for production using, but abandoned (issue https://github.com/lmachens/meteor-apm-server/issues/47 ).

I have a fork with some improvements, but i'd like to merge it into @Imachens repo.
 
 ## Progress
 - [ ] Discussion finished
 - [ ] Moved / Forked
 - [ ] Team Created
 - [ ] Meteor Org Created and set as maintainer

**Comment by vparpoil** [comment]
We have been using it as well. I think we should also migrate the client repo : https://github.com/lmachens/kadira

**Comment by afrokick** [comment]
Yes, i created a separate issue for that https://github.com/Meteor-Community-Packages/organization/issues/38 for my fork of Imachens:kadira. But i can merge my fix into it if it more preferable.

**Comment by afrokick** [comment]
@Imachens what do you think?

**Comment by afrokick** [comment]
@lmachens what do you think?

**Comment by lmachens** [comment]
I found some maintainers for the project, so it is not abandoned. Anyway, how can I help?

**Comment by SimonSimCity** [comment]
I think it's worth mentioning as a note aside that there's also a good alternative to Kadira: the ELK stack. You might want to use it anyways if you're managing the infrastructure yourself - but it can be used equally easy for a hosted meteor installation: https://github.com/kschingiz/meteor-elastic-apm

**Comment by evolross** [comment]
Could one run Meteor APM and Elastic APM at the same time?

**Comment by afrokick** [comment]
I use Elastic only for logs. But don't like it. It consumes a huge amount of RAM.

**Comment by evolross** [comment]
When you say it consumers a lot of RAM - like just having the connection to Elastic (running on another machine) uses a lot of RAM? Or are you running Elastic on the same machine as your app and thus is uses a lot of RAM?

Could you explain more about your configuration. As I've been increasingly unsatisfied by Galaxy's logging and was going to set up the Elastic integration.

**Comment by SimonSimCity** [comment]
@evolross I haven't tried running them side-by-side. I was using a self-hosted version of Kadira (don't recall how we did it - I just now found tons of tutorials online which you can try out) which at some point in time got really slow - could well be that it would've been OK again with 5min of work - but we also wanted more NodeJS internal and also infrastructural metrics of our systems, so we did a hard cut. One metric we desperately were missing is the event-loop-delay.
